tests/Feature/Repositories/TaskRepositoryTest.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Tests\Feature\Repositories;
+
+use App\Models\Task;
+use App\Models\User;
+use App\Repositories\CommentRepository;
+use App\Repositories\TaskRepository;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Support\Collection;
+use Tests\TestCase;
+
+class TaskRepositoryTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function test_create_task_is_successful(): void
+    {
+        $user = new User();
+        $user->setAttribute('name','test');
+        $user->setAttribute('email', 'test@gmail.com');
+        $user->setAttribute('password', '2233');
+        $user->save();
+
+        $repository = new TaskRepository;
+
+        $result = $repository->createTask([
+            'name' => 'task',
+            'description' => 'task one',
+            'status' => 'ongoing'
+        ]);
+
+        $repository2 = new CommentRepository;
```

Comment:
`TaskRespositoryTest` should test only `TaskRepository` not `CommentRepository`. So test only one class/method at a time.

Reply:
```
$taskRespository
```